llvm/lib/CodeGen/MachineLICM.cpp

```diff
@@ -423,10 +423,47 @@ static bool InstructionStoresToFI(const MachineInstr *MI, int FI) {
   return false;
 }
 
+static void applyBitsNotInRegMaskToRegUnitsMask(const TargetRegisterInfo *TRI,
```

Comment:
const ref.

This whole thing should probably be a helper somewhere else, but it's OK to move it in a later commit.